meetings/2022-07/jul-21.md

```diff
@@ -0,0 +1,795 @@
+
+JSC: All right, next up is WMS no need to talk saying. Moving forward to policy map since its proposal can unlock better userland Solutions. Thanks WMS. I would say would say again that Although policy maps and would make this much easier, it would still be a big hassle to do deal with multiple arguments as well as as this receiver like you would at the like DE mentioned you would meet nested Maps or in this case, I guess nested policy maps, that's not like with nested policies. That sounds, I mean that is something that is probably very narrow namik. Having that, yes, it is theoretically possible just very on are gonna make even with policy Max unless you unless if you want to do with multiple arguments now the queue is empty. I would like to ask again for whether we want to explore this problem space given what we've presented or the if anyone would not support this for stage 1.
+
+WH: I don't know how to translate this into Stage 1 or not, but what I'd like to see as a follow-up is a good analysis of existing practice. So I definitely want to explore the problem space, starting with gathering data on whether this is best done in userland or via a language-defined feature. The only hang up I have with Stage 1 is that Stage 1 implies that it's something that's we’re exploring within an eye of adding it to the language, and I don't think we know that yet with this.
+
+JSC: now, to be clear, the formal definition of stage one acceptance signifies that the committee expects to devote time to examining the problem space in Solutions. Now it is true that there is a connotation that this has been elevated to something that they are consistently adding to the language formally. It's not in it. The in the definition, it's just a good time. Having said that Um, I would like to ask again since we're at time. Do we have consensus for stage 1 to devote time to examine the problem space for which is to come back with more prior art
+
+RPR: and SFC here states that the proposal does make TG2 requirements for stage 1.
+
+DE: I support stage 1
+
+RKG: I also support stage 1
+
+RPR: And we'll in WMS says that he supports further investigations on the problem space. okay, so final final call are there any objections to stage one? Okay, no objections. Congratulations. You have stage 1.
+
+JSC: Thank you Very much. I look forward to collecting every single prior art of memorization in existence.
+
+RPR: Strong ending.
+
+### Conclusion/Resolution
+
+- Stage 1
+
+## Ergonomic Dynamic Object Restructuring
+
+Presenter: Hemanth HM (HHM)
+
+- [proposal](https://github.com/tc39-transfer/proposal-object-pick-or-omit)
+- [slides]()
```

Comment:
It seems the link to the slides is missing

Reply:
There was no slide link on the agenda. This is for @hemanth to fix up later.